src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const remProp of action.keysToRemove) {
```

Comment:
I would give a name "key" in case you take `key` in `keysToRemove`
```suggestion
        for (const key of action.keysToRemove) {
```